app/views/layouts/application.html.erb

```diff
@@ -12,5 +12,8 @@
 
   <body>
     <%= yield %>
+    <% if flash[:alert].present? %>
+        <p class="my_alert"><%= flash[:alert] %></p>
+    <% end %>
```

Comment:
Normally you want flash errors to be the first thing displayed on the page. Otherwise, these important messages may get hidden underneath the regular page content.

```suggestion
    <% if flash[:alert].present? %>
        <p class="my_alert"><%= flash[:alert] %></p>
    <% end %>
    <%= yield %>
```

You don't need to do this, but I'll see if I can update the lesson to emphasize this for the next class.

Reply:
Tank you for advice.